update/release.go

```diff
@@ -54,10 +54,11 @@ type Release struct {
 // NB: these get sent from fluxctl, so we have to maintain the json format of
 // this. Eugh.
 type ReleaseSpec struct {
-	ServiceSpecs []ServiceSpec
-	ImageSpec    ImageSpec
-	Kind         ReleaseKind
-	Excludes     []flux.ServiceID
+	ServiceSpecs  []ServiceSpec
+	ContainerName string
```

Comment:
Could we have some commentary on the meaning and use of this field please? E.g., what happens if the ServiceSpec is `<all>`, and the ImageSpec is `<latest>` (or whatever those literals are).